Library/Homebrew/bottle_version.rb

```diff
@@ -63,3 +65,25 @@ def self._parse spec
     super
   end
 end
+
+class BottlePkgVersion < PkgVersion
```

Comment:
I guess it's unclear to me just from the code when I would use one over the other.